pkg/virt-api/rest/subresource.go

```diff
@@ -609,15 +661,20 @@ func (app *SubresourceAPIApp) StartVMRequestHandler(request *restful.Request, re
 				v1.VirtualMachineStateChangeRequest{Action: v1.StopRequest, UID: &vmi.UID},
```

Comment:
I think the `needsRestart == true` case can also have the paused flag (conditionally) added, right?
For example, if you `virtctl start --paused` a manually-controlled VMI which currently parks at Succeeded state, I'd expect it to indeed start paused.

Reply:
Added. Thank you